addons/account/views/account_report.xml

```diff
@@ -3,16 +3,14 @@
     <data>
         <!-- QWeb Reports -->
         <record id="account_invoices" model="ir.actions.report">
-            <field name="name">Invoices</field>
+            <field name="name">PDF</field>
             <field name="model">account.move</field>
             <field name="report_type">qweb-pdf</field>
             <field name="report_name">account.report_invoice_with_payments</field>
             <field name="report_file">account.report_invoice_with_payments</field>
             <field name="is_invoice_report">True</field>
             <field name="print_report_name">(object._get_report_base_filename())</field>
             <field name="attachment"/>
-            <field name="binding_model_id" ref="model_account_move"/>
-            <field name="binding_type">report</field>
```

Comment:
Why removing those values? This doesn't look correct.
At least force the default values for updates.

Reply:
Because I don't want this action_report to be available through the Print menu :thinking: 
According to [our documentation](https://www.odoo.com/documentation/master/developer/reference/backend/actions.html#report-actions-ir-actions-report):
> If you define your report through a `<record>` instead of a `<report>` tag and want the action to show up in the Print menu of the model’s views, you will also need to specify binding_model_id from [Bindings](https://www.odoo.com/documentation/master/developer/reference/backend/actions.html#reference-bindings). It’s not necessary to set binding_type to report, since ir.actions.report will implicitly default to that.

Reply:
So you need to force an empty value
```xml
<field name="binding_model_id" eval="False"/>
```